### DISCUSSION TITLE
Hang at launch, Transmission-f2aa3e197b Build #9024 (Apr 18, 2023, 9:33:50 PM)

### DISCUSSION THREAD

**Discussion by dantearmok** [transmission/transmission#5417] [Support]
Mac mini (late 2012, 2.5-GHz 4-Core i5), 16GB RAM, plus an assortment of USB & FW HDs.
macOS 10.15 Catalina, ClamXAV, PeakHour, PIA, and Xcode 12.3 (all fully updated).

Transmission-f2aa3e197b Build #9024 (Apr 18, 2023, 9:33:50 PM)

Launched Transmission, went thru all the normal OS dialogs.  Transmission never brought up any windows.  Clicking on its dock icon did nothing.  Activity Monitor says it's Not Responding.

Sample attached...
[Sample of Transmission, on launch.txt.zip](https://github.com/transmission/transmission/files/11267312/Sample.of.Transmission.on.launch.txt.zip)

**Comment by ckerr** [comment]
Possibly fixed by 3044c14b106fb3131212bae39c3e15345d6966a4. If not, please file a bug report in [Issues](https://github.com/transmission/transmission/issues). Thanks!

Locking discussion.
